docs/web-component-samples/tsconfig.json

```diff
@@ -3,9 +3,11 @@
     "compilerOptions": {
         "outDir": "./dist",
         "declaration": false,
-        "declarationDir": null
+        "declarationDir": null,
+        "moduleResolution": "NodeNext",
+        "module": "NodeNext"
```

Comment:
Little context- https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-7.html
When we are using export in package.json we need smarter way of compilling so that it picks up the types.